tests/OSPSuite.Core.Tests/Domain/ModelCoreSimulationExtensionsSpecs.cs

```diff
@@ -33,7 +33,7 @@ protected override void Context()
          reactionBuildingBlock.Add(reactionAtoB);
          reactionBuildingBlock.Add(reactionBtoA);
 
-         var module = new Module {Reactions = reactionBuildingBlock};
+         var module = new Module {reactionBuildingBlock};
```

Comment:
Beausse I implement the IEnumrable pattern with an ADd method,, you can Add bb like this